public/index.html

```diff
@@ -68,7 +76,7 @@
       body {
         margin: 0;
         padding: 0;
-        font-family: regular;
+        font-family: "Raleway", sans-serif;
```

Comment:
If we're going to only use Raleway font across entire or majority of website 
Add *{
font-family: "Raleway", sans-serif;
}

Then no need to add it individually for each classNames